cmd/mount.go

```diff
@@ -108,32 +108,8 @@ func (opt *mountOptions) validate(skipNonEmptyMount bool) error {
 	if _, err := os.Stat(opt.MountPath); os.IsNotExist(err) {
 		return fmt.Errorf("mount directory does not exist")
 	} else if common.IsDirectoryMounted(opt.MountPath) {
-		// Try to cleanup the stale mount
-		log.Info("Mount::validate : Mount directory is already mounted, trying to cleanup")
```

Comment:
This code shall not be removed, if there are bugs in this logic fix it. Basically we want to check whether the existing mount has an active process associated with it or not. If mount exists but respective blobfuse2 process has exited, creating a dangling mount then cleanup shall run otherwise it shall just fail saying its already mounted.